onnx/funconnx/aionnx/op_argmax.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=W0221
+
+import numpy as np  # type: ignore
+
+from ...defs import onnx_opset_version
+from ._op import OpRunArg
+
+
+def _argmax(data, axis=0, keepdims=True):  # type: ignore
+    result = np.argmax(data, axis=axis)
+    if keepdims and len(result.shape) < len(data.shape):
+        result = np.expand_dims(result, axis)
+    return result.astype(np.int64)
+
+
+def _argmax_use_numpy_select_last_index(data, axis=0, keepdims=True):  # type: ignore
+    data = np.flip(data, axis)
+    result = np.argmax(data, axis=axis)
+    result = data.shape[axis] - result - 1
+    if keepdims:
+        result = np.expand_dims(result, axis)
+    return result.astype(np.int64)
+
+
+class _ArgMax(OpRunArg):
+    def _run(self, data, axis=None, keepdims=None):  # type: ignore
+        axis = axis or self.axis  # type: ignore
```

Comment:
Trying to understand this: is the `or self.axis` part necessary? At least, the calls to _run I saw in some part of the code I saw (in op_run.py or proto_run.py) already passed in self.axis.

Reply:
Remove, it is not necessary. This code was moved in OpRun.